tomatoTomato/Models/tomatoTaskModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  tomatoTaskModel.swift
+//  tomatoTomato
+//
+//  Created by Maria Kharybina on 17/11/2022.
+//
+
+import Foundation
+
+//struct tomatoTaskModel: Codable {
```

Comment:
please don't commit commented code. One can run git diff to go through the history if necessary.